Consul.Test/CatalogTest.cs

```diff
@@ -238,5 +265,57 @@ public async Task Catalog_EnableTagOverride()
                 Assert.True(services.Response[0].ServiceEnableTagOverride);
             }
         }
+
+        [SkippableFact]
+        public async Task Catalog_ServicesForNodes()
+        {
+            var cutOffVersion = SemanticVersion.Parse("1.7.0");
+            Skip.If(AgentVersion < cutOffVersion, $"Current version is {AgentVersion}, but `logjson` is only supported from Consul {cutOffVersion}");
```

Comment:
Please update the message to mention the actual feature which is not supported (instead of `logjson`)

Reply:
ooh my bad